packages/ui5-application-writer/package.json

```diff
@@ -32,6 +32,7 @@
         "!dist/**/*.map"
     ],
     "dependencies": {
+        "@sap-ux/eslint-plugin-fiori-tools": "workspace:*",
```

Comment:
Is this dependency required?

Reply:
Good catch. It should have been a devDep for the tests. Fixed in https://github.com/SAP/open-ux-tools/pull/984/commits/513968ea97a7c539a82c59748a194baf37f8e103